httpx/dispatch/http2.py

```diff
@@ -96,6 +96,7 @@ def init_complete(self) -> BaseEvent:
         del self.state.local_settings[h2.settings.SettingCodes.ENABLE_CONNECT_PROTOCOL]
 
         self.state.initiate_connection()
+        self.state.increment_flow_control_window(2 ** 24)
```

Comment:
Our of curiosity, what’s the default value for the window?

Reply:
65535

Reply:
Okay. Should this line require a comment of any sort? It's the kind of line I'd bump into and wonder "Why?".